tutorial/.devcontainer/Dockerfile

```diff
@@ -0,0 +1,30 @@
+# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.187.0/containers/alpine/.devcontainer/base.Dockerfile
+
+# [Choice] Alpine version: 3.13, 3.12, 3.11, 3.10
+ARG VARIANT="3.13"
```

Comment:
I was playing with a variation of this to create a variation of this for Z80 assembly and had to switch to 3.12  when attempting to build the docker container on a Chromebook. Apparently this is a known issue with 3.13 when hosted in some linux distros.